config/initializers/pp50_insitituions.rb

```diff
@@ -0,0 +1,13 @@
+require "csv"
+schools = CSV.table("config/data/202405XX_Schools-disadvantage-lists.csv")
+pp50_schools = schools.each_with_object({}) { |school, hash| hash[school[:urn].to_s] = true }
+fe = CSV.table("config/data/pp50further_education.csv")
+fe_ukprn = fe.each_with_object({}) { |school, hash| hash[school[:ukprn].to_s] = true }
+ey = CSV.table("config/data/early-years-all-settings.csv")
+ey_ofsted_urn = ey.each_with_object({}) { |school, hash| hash[school[:ofsted_urn].to_s] = true }
+childminders = CSV.table("config/data/childminders.csv")
+childminders_urn = childminders.each_with_object({}) { |school, hash| hash[school[:ofsted_urn].to_s] = true }
+PP50_SCHOOLS_URN_HASH = pp50_schools
+PP50_FE_UKPRN_HASH = fe_ukprn
+EY_OFSTED_URN_HASH = ey_ofsted_urn
+CHILDMINDERS_OFSTED_URN_HASH = childminders_urn
```

Comment:
Why do it like this and not use SQL tables?

Reply:
This is static data, and I find it easier that way. In theory, we could add flags in the SQL, but this way is faster. It could be an improvement later and `pp50_institutions.rb` could become a base for script that loads data to SQL.

Reply:
Just feels like a lot of data to live in memory. I suppose it's fine this way for now.

Reply:
I echo @TBenedyk's thoughts here.

In addition to the additional memory we also need to keep an eye on the 'sustainability' of the project - if in 5 years time there are 0 devs, could it be operated?

Can we have a ticket near the top of the backlog to
* move this into the database
* update the GIAS importer to set any additional fields when it runs
* or, if the data isn't in the GIAS export add an admin page where a CSV can be uploaded and processed (maybe - open to any nicer solution than this)

Reply:
Also it feels a little funny that they're hashes only with `true`. Would an array not make more sense, then you can check with:

```ruby
"123456".in?(pp50_schools)
```